internal/backend/dispatcher/workflow.go

```diff
@@ -3,7 +3,6 @@ package dispatcher
 import (
 	"errors"
 	"fmt"
-	"maps"
```

Comment:
GitHub says the variables below are unused.

Reply:
huh i don't see it in my repo, probably not most up to date